MMM-COVID19.js

```diff
@@ -24,12 +24,17 @@ Module.register("MMM-COVID19", {
     headerRowClass: "small", // small, medium or big
     infoRowClass: "big", // small, medium or big
     updateInterval: 300000, // update interval in milliseconds
-    fadeSpeed: 4000
+    fadeSpeed: 4000, 
+	timeFormat:"D.M. HH.mm"
   },
 
   getStyles: function() {
     return ["MMM-COVID19.css"]
   },
+  
+  getScripts: function () {
+    return ['moment.js']
+  },
```

Comment:
@hildwin there is no need to add this here, because this file is available in the vendor folder (https://docs.magicmirror.builders/development/core-module-file.html#getscripts)

Please remove the whole function.